### ISSUE TITLE
Not working feel when using JS Bookmark snippet

### ISSUE THREAD

**Issue by surjithctly** [Khan/tota11y#22]
I've added the snippet to my bookmarks bar. Then clicked on it. But nothing happened. Tried on another page. Same. After 10 mins, I discovered the small glass icon in the bottom left corner and that's easy to miss. Mainly for the First Time users. 

Its better to Open and activate the First comment first. But only for Bookmarked users, not for those who inserts the <script> version.

**Comment by jdan** [comment]
Good idea. I may have to leak a `tota11y` global variable so I can things like `tota11y.open("contrast")`.

Thanks for suggesting this, sorry you had trouble finding it!

**Comment by brownhead** [comment]
I think the `tota11y` global would be useful. Different installation could even use it to automatically enable the plugins that particularly useful to them.

This could also be an interesting way of addressing #12. You could expose the number of errors through a function, and then different installations could automatically check/enable them if there's errors.
